lib/streamlit/runtime/caching/cache_errors.py

```diff
@@ -22,22 +21,7 @@
     StreamlitAPIException,
     StreamlitAPIWarning,
 )
-
-
-class CacheType(enum.Enum):
-    """The function cache types we implement."""
-
-    DATA = "DATA"
-    RESOURCE = "RESOURCE"
-
-
-def get_decorator_api_name(cache_type: CacheType) -> str:
-    """Return the name of the public decorator API for the given CacheType."""
-    if cache_type is CacheType.DATA:
-        return "cache_data"
-    if cache_type is CacheType.RESOURCE:
-        return "cache_resource"
-    raise RuntimeError(f"Unrecognized CacheType '{cache_type}'")
```

Comment:
`legacy_caching` needed access to the `CacheType` enum. I moved the enum into its own module (`cache_type.py`) so that legacy_caching could import it without getting all of `cache_errors.py`